### ISSUE TITLE
Thank you! What are the differences in dithering options for 10-bit output?

### ISSUE THREAD

**Issue by Carl-Robert** [ledoge/novideo_srgb#15]
First of all, not a real issue!

EDID primaries work just fine without access to a colorimeter on my DELL S3220DGF monitor! Just had to delete the manufacturer provided .icm profile (or things got "corrected" twice in color managed applications). The EDID values and the .icm profile looked pretty identical when eyeballing, but of course clamping this way benefits ALL programs.

However I am not sure what dithering option is the "best"? Not sure if I see any difference, but would be nice to know which in theory is the most accurate. In my case my monitor is 1440p 10-bit panel. Or should 10-bit users disable dithering altogether?

**Comment by aufkrawall** [comment]
Dithering is enabled by default in the Nvidia driver for 10 bit output. If you don't notice additional banding with the clamp active, there is no need to act.

**Comment by ledoge** [comment]
I agree – I'm not sure how high the internal precision is, i.e. if enabling dithering at 10 bit even has a benefit, but leaving it off should be perfectly fine.

**Comment by Carl-Robert** [comment]
Thanks for the answers! I guess I'll squint a bit more between disabled, temporal and default dithering. (I don't think I'll notice anything though!)


The reason for asking is because with so many options and no documentation about them makes it hard for the end user to know the difference between the different dithering methods. So would be nice to have some "official" recommendation/theoretically the best performing dithering method mentioned (and if there is such a thing, could cut the less performant options). Searching the web for these terms brings just confusion :)

**Comment by markanini** [comment]
Open the gradient test from here https://www.eizo.be/monitor-test/ For me temporal is the only effective option.
